File: app/src/main/java/com/example/myanimedata/api/AnimeResponseDetail.java
package com.example.myanimedata.api;

import com.google.gson.annotations.SerializedName;

public class AnimeResponseDetail {
    @SerializedName("data")
    private final AnimeDetail animeDetails;

    public AnimeDetail getAnimeDetails() {
        return animeDetails;
    }

    public AnimeResponseDetail(AnimeDetail animeDetails) {
        this.animeDetails = animeDetails;
    }
}

File: app/src/main/java/com/example/myanimedata/api/AnimeResponseDetail.kt
package com.example.myanimedata.api

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.parcelize.Parcelize

@Parcelize
data class AnimeResponseDetail(
    @field:SerializedName("data") val animeDetails: AnimeDetail? = null
) : Parcelable